src/index.js

```diff
@@ -0,0 +1,159 @@
+const weatherUrl = 'http://127.0.0.1:5000/weather';
+
+const increaseTemp = () => {
+    const tempContainer = document.querySelector('#tempValue');
+    state.temperature++;
+    tempContainer.textContent = state.temperature;
+    changeTempColor();
+    changeLandscape();
+};
+
+const decreaseTemp = () => {
+    const tempContainer = document.querySelector('#tempValue');
+    state.temperature--;
+    tempContainer.textContent = state.temperature;
+    changeTempColor();
+    changeLandscape();
+};
+
+const renameCityHeader = () => {
+    const headerCity = document.querySelector('#headerCityName');
+    const newCity = document.querySelector('#cityNameInput');
+    state.city = newCity.value;
+    headerCity.innerText = state.city;
+};
+
+const resetCity = () => {
+    const headerCity = document.querySelector('#headerCityName');
+    headerCity.innerText = state.defaultCity;
+    const newCity = document.querySelector('#cityNameInput');
+    newCity.value = state.defaultCity;
+};
+
+const getLatAndLon = () => {
+    axios.get(locationUrl, {
+        params: {
+            q: state.city,
+            format: "json",
+        }
+    })
+        .then((res) => {
+            state.lat = res.data[0].lat;
+            state.lon = res.data[0].lon;
+        })
+        .then(getCurrentWeather())
+        .catch((err) => {
+            console.log("Failed to load latitude and longitude")
+        })
+};
+
+const tempConvertKtoF = (temp) => (temp - 273.15) * (9 / 5) + 32;
+
+const setTempToRealTimeTemp = () => {
+    const tempContainer = document.querySelector('#tempValue');
+    tempContainer.innerText = state.temperature;
+    changeTempColor();
+    changeLandscape();
+};
+
+const getCurrentWeather = () => {
+    axios.get(weatherUrl, {
+        params: {
+            lat: state.lat,
+            lon: state.lon,
+        }
+    })
+        .then((res) => {
+            state.temperature = res.data.main.temp;
+            state.temperature = Math.round(tempConvertKtoF(state.temperature));
+        })
+        .then(setTempToRealTimeTemp)
+        .catch((err) => {
+            console.log("Unable to get real time temperature.")
+        })
+};
+
+const changeSky = () => {
+    const skyContainer = document.querySelector('#sky');
+    const skySelectInput = document.querySelector('#skySelect')
+    const skyInputValue = skySelectInput.value;
+    if (skyInputValue in skyState) {
+        skyContainer.innerText = skyState[skyInputValue];
+    }
+};
+
+const changeLandscape = () => {
+    const landscapeContainer = document.querySelector('#landscape');
+
+    landscapeContainer.innerText = landState[
+        state.temperature >= 80 ? "80+" : 
+        state.temperature >= 70 ? "70+" : 
+        state.temperature >= 60 ? "60+" : 
+        "59-"
+    ];
+};
+
+const changeTempColor = () => {
+    const tempContainer = document.querySelector('#tempValue');
+    
+    tempContainer.style.color = tempState[
```

Comment:
`element.style` is the JS equivalent of setting an inline style on a tag, which we generally want to avoid. Prefer using the `className` or `classList` attributes to modify the CSS classes associated with an element instead. This provides much more flexibility, as we can change many style properties simply by updating the CSS rules.